dotnet/Cucumber.HtmlFormatter/Cucumber.HtmlFormatter.csproj

```diff
@@ -19,11 +19,11 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <EmbeddedResource Include="Resources\**\*"/>
+    <EmbeddedResource Include="Resources\**\*" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cucumber.Messages" Version="[27.0, 28.0)"/>
```

Comment:
I think it would make sense to keep the version range here.

Reply:
In which case, I think you intended the range to be [27.0, 29.0), correct?  {Greater than or equal to 27 but less than 29}

Reply:
That is correct!